app/frontend/src/submissions/components/SubmissionForm/Location.vue

```diff
@@ -285,6 +302,97 @@ export default {
       this.streetAddressInput = String(this.ownerMailingAddress)
       this.cityInput = String(this.ownerCity)
     }
+  },
+  methods: {
+      /**
+     * Asynchronously fetches address suggestions based on the owner's address input.
+     * If no input is provided, it clears the current suggestions.
+     * On success, it maps the received data to full addresses and updates the addressSuggestions state.
+     * On failure, it logs the error and clears the current suggestions.
+     * Finally, sets the loading state to false.
+     */
+    async fetchAddressSuggestions() {
```

Comment:
I use determine a minimum length of characters (3?) before hitting the API with requests. 
You likely won't get useful feedback with a low character count, and its generating more network traffic from the user device to the api

Reply:
implemented